js-client-sdk.api.md

```diff
@@ -216,6 +216,9 @@ export interface IPrecomputedClientConfigSync {
     throwOnFailedInitialization?: boolean;
 }
 
+// @public
+export type LogLevel = 'trace' | 'debug' | 'info' | 'warn' | 'error' | 'silent';

```

Comment:
We're not supporting `'fatal'` in [this list](https://getpino.io/#/docs/api?id=level-string)?